Makefile

```diff
@@ -75,6 +79,13 @@ build:
 generate: $(_GENERATE_DEPS_EXECUTABLES)
 	PATH=$(_GENERATE_DEPS_DIR):$$PATH ./scripts/generate.sh
 
+.PHONY: protogenerate
+protogenerate: $(_GENERATE_DEPS_EXECUTABLES)
+	@echo "TODO: merge with make generate once apache thrift issues fixed"
```

Comment:
That's the other issue I'm working on right now, I might have another PR up for it today, it's the `set<T>` generating issue